tests/lsdb_macauff/import_pipeline/test_run_import.py

```diff
@@ -0,0 +1,73 @@
+import os
+
+import pytest
+from hipscat.catalog.association_catalog.association_catalog import AssociationCatalog
+from hipscat.io import file_io
+from hipscat_import.catalog.file_readers import CsvReader
+
+import lsdb_macauff.import_pipeline.run_import as runner
+from lsdb_macauff.import_pipeline.arguments import MacauffArguments
+from lsdb_macauff.import_pipeline.convert_metadata import from_yaml
+
+# pylint: disable=too-many-instance-attributes
+# pylint: disable=duplicate-code
+
+
+@pytest.mark.dask
+def test_bad_args(dask_client):
+    """Runner should fail with empty or mis-typed arguments"""
+    with pytest.raises(TypeError, match="MacauffArguments"):
+        runner.run(None, dask_client)
+
+    args = {"output_artifact_name": "bad_arg_type"}
+    with pytest.raises(TypeError, match="MacauffArguments"):
+        runner.run(args, dask_client)
+
+
+@pytest.mark.dask
+def test_object_to_object(
+    import_catalog_a,
+    import_catalog_b,
+    tmp_path,
+    import_metadata_yaml,
+    import_match_csv,
+    dask_client,
+):
+    """Test that we can create a MacauffArguments instance with two valid catalogs."""
```

Comment:
Should this docstring be something along the lines of:
> Test that we can create an association catalog by running the Macauff import pipeline

Reply:
Yes - and feel free to use "suggestions" on PRs, particularly for tweaking wording here and there.